BUILD

```diff
@@ -370,6 +370,7 @@ grpc_cc_library(
         "src/cpp/common/tls_credentials_options_util.h",
         "src/cpp/server/secure_server_credentials.h",
     ],
+    defines = ["GRPC_XDS_USER_AGENT_SUFFIX=C++"],
```

Comment:
How did this get in here?  It doesn't seem to have anything to do with the rest of this PR, and I don't think it's actually useful for anything.  We aren't planning to set the xDS user agent suffix for C++ at all, and even if we were, this is the wrong macro name.

Can you please remove this?  Thanks!

Reply:
CC @apolcyn

Reply:
huh! I've no idea how this got in. https://github.com/grpc/grpc/pull/26486

Reply:
I thought this was intentional for the user agent work we were planning to get into the previous release.